src/Composer/ComposerEnvironment.php

```diff
@@ -107,11 +108,25 @@ public function checkFilePermissions()
             $this->getTemporaryVendorPath(),
         ];
 
+        $paths = [];
+
         foreach ($pathsToCheck as $path) {
             if (file_exists($path) && !is_writable($path)) {
-                throw new FilePermissionException('Write permission missing for '.$path);
+                $paths[] = str_replace(base_path(), '/', $path);
             }
         }
+
+        return $paths;
+    }
+
+    /**
+     * @throws FilePermissionException
+     */
+    public function checkFilePermissions()
+    {
+        foreach ($this->retrieveFilePermissions() as $path) {
+            throw new FilePermissionException('Write permission missing for '.$path);
```

Comment:
I don't think so, it's obvious which paths are needed. We can even document them, logging absolute paths for such a thing is simply unnecessary because these would be logged with every single hit in admin.